pycode/examples/secir_groups.py

```diff
@@ -52,55 +54,54 @@ def run_secir_groups_simulation():
+        model.parameters.InfectiousTimeMild[AgeGroup(i)] =  6.  # 4-14  (=R4^(-1))
+        model.parameters.SerialInterval[AgeGroup(i)] = 4.2   # 4-4.4 // R_2^(-1)+0.5*R_3^(-1)
+        model.parameters.HospitalizedToHomeTime[AgeGroup(i)] = 12.  # 7-16 (=R5^(-1))
+        model.parameters.HomeToHospitalizedTime[AgeGroup(i)] = 5.  # 2.5-7 (=R6^(-1))
+        model.parameters.HospitalizedToICUTime[AgeGroup(i)] = 2.  # 1-3.5 (=R7^(-1))
+        model.parameters.ICUToHomeTime[AgeGroup(i)] = 8.  # 5-16 (=R8^(-1))
+        model.parameters.ICUToDeathTime[AgeGroup(i)] = 5.  # 3.5-7 (=R5^(-1))
 
         # Initial number of peaople in each compartment
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Exposed)] = 100
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Carrier)] = 40
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Infected)] = 80
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Hospitalized)] = 40
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.ICU)] = 20
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Recovered)] = 7
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Dead)] = 3
-        model.populations.set_difference_from_group_total_AgeGroup((secir.AgeGroup(i), 
-            secir.Index_InfectionState(secir.InfectionState.Susceptible)), populations[i])
+        model.populations[AgeGroup(i), Index_InfectionState(State.Exposed)] = 100
+        model.populations[AgeGroup(i), Index_InfectionState(State.Carrier)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Infected)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Hospitalized)] = 20
+        model.populations[AgeGroup(i), Index_InfectionState(State.ICU)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Recovered)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Dead)] = 0
+        model.populations.set_difference_from_total((AgeGroup(i), Index_InfectionState(State.Susceptible)), populations[i])
 
         # Compartment transition propabilities
-        model.parameters.probabilities[i].set_infection_from_contact(1.0)
-        model.parameters.probabilities[i].set_carrier_infectability(0.67)
-        model.parameters.probabilities[i].set_asymp_per_infectious(0.09)
-        model.parameters.probabilities[i].set_risk_from_symptomatic(0.25)
-        model.parameters.probabilities[i].set_hospitalized_per_infectious(0.2)
-        model.parameters.probabilities[i].set_icu_per_hospitalized(0.25)
-        model.parameters.probabilities[i].set_dead_per_icu(0.3)
-        
+           
+        model.parameters.RelativeCarrierInfectability[AgeGroup(i)] = 0.67  #TODO: must this parameter be set? 
```

Comment:
No, not necessarily. As all parameters, it has a default value. But it is better to explicitly call the setters for all our parameters in the example. Please remove comment.